src/helpers/permit.ts

```diff
@@ -11,11 +11,8 @@ const PERMIT2_ADDRESS = "0x000000000022D473030F116dDEE9F6B43aC78BA3"; // same on
 export type Permit = {
   id: number;
   createdAt: Date;
-  organizationId: number | null;
-  repositoryId: number;
-  issueId: number;
+  updatedAt: Date;
   networkId: number;
-  bountyHunterId: number;
   bountyHunterAddress: string;
```

Comment:
Not sure if it's appropriate to handle these in this pool request, but we should rename every instance of "bounty hunter" into "contributor" or "assignee" to be specific with our terminology

Reply:
> Not sure if it's appropriate to handle these in this pull request, but we should rename every instance of "bounty hunter" into "contributor" or "assignee" to be specific with our terminology

I think that is a lot of change for this issue